ui/js/component/transactionList/internal/TransactionListItem.jsx

```diff
@@ -80,6 +92,12 @@ class TransactionListItem extends React.PureComponent {
         <td>
           <LinkTransaction id={txid} />
         </td>
+        <td>
```

Comment:
I'm inclined to make this an icon next to either the amount or the type.

Also, since this logic makes both received tips, supports, and publishes revokable, I think we need to consider the way we present this in each case.

For a tip someone else sent me, this should be presented as something like an "unlock", with a warning that it will reduce the committed credits to that URL.

For a claim I made, or a support I made to someone else's content, this is more of a delete action.

Reply:
I've wrote some placeholder texts in the new commit, as draft. I was hoping you could assist me in what will be the final modal say.

Reply:
I'll make any copy changes before I merge.